docs/en/ingest-management/elastic-agent/grant-access-to-elasticsearch.asciidoc

```diff
@@ -0,0 +1,146 @@
+[[grant-access-to-elasticsearch]]
+= Grant standalone {agent}s access to {es}
+
+You can use either API keys or user credentials to grant standalone {agent}s
+access to {es} resources. The following minimal permissions are required to send
+logs, metrics, traces, and synthetics to {es}:
+
+* `monitor` cluster privilege
+* `auto_configure` and `create_doc` index privileges on `logs-*`, `metrics-*`,
```

Comment:
`logs-*-*` is required. I know we still use `logs-*` in many places and it works too but we likely to reduce it to just `logs-*-*` if possible. Same for the other patterns.